sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskJurisdictionPredicateValidator.java

```diff
@@ -61,18 +64,41 @@ public static TaskJurisdictionPredicateValidator of(TaskQueryContext qc, User us
 
 	@Override
 	public Predicate isRootInJurisdictionOrOwned() {
-		final Predicate createdByCurrentUser = cb.and(cb.isNotNull(joins.getCreator()), cb.equal(joins.getCreator().get(User.UUID), user.getUuid()));
+		final Predicate createdByCurrentUser = getCreatedByCurrentUser();
+		final Predicate assignedToCurrentUser = getAssignedToCurrentUser();
+		return cb.or(createdByCurrentUser, assignedToCurrentUser, isRootInJurisdiction());
+	}
 
+	private Predicate getAssignedToCurrentUser() {
 		final Predicate assignedToCurrentUser =
 			cb.and(cb.isNotNull(joins.getAssignee()), cb.equal(joins.getAssignee().get(User.UUID), user.getUuid()));
-		return cb.or(createdByCurrentUser, assignedToCurrentUser, isRootInJurisdiction());
+		return assignedToCurrentUser;
+	}
+
+	private Predicate getCreatedByCurrentUser() {
+		return cb.and(cb.isNotNull(joins.getCreator()), cb.equal(joins.getCreator().get(User.UUID), user.getUuid()));
 	}
 
 	@Override
 	public Predicate isRootInJurisdiction() {
 		return isInJurisdictionByJurisdictionLevel(user.getJurisdictionLevel());
 	}
 
+	@Override
+	public Predicate isRootInJurisdictionForRestrictedAccess() {
+		final Predicate createdByCurrentUser = getCreatedByCurrentUser();
+		final Predicate assignedToCurrentUser = getAssignedToCurrentUser();
+
+		List<Predicate> associationPredicates = new ArrayList<>(associatedJurisdictionValidators.size());
+		for (JurisdictionValidator<Predicate> associatedJurisdictionValidator : associatedJurisdictionValidators) {
+			if (!TravelEntryJurisdictionPredicateValidator.class.isAssignableFrom(associatedJurisdictionValidator.getClass())
```

Comment:
this if condition seems to be useless, those kind of jurisdiction validators are not used for tasks

Reply:
removed